src/agent/debuglet.ts

```diff
@@ -322,6 +349,32 @@ export class Debuglet extends EventEmitter {
   }
 
   /**
+   * isReady returns a promise that only resolved if the last breakpoint update
+   * happend within a duration (PROMISE_RESOLVE_CUT_OFF_IN_MILLISECONDS). This
+   * feature is mainly used in Google Cloud Function (GCF), as it is a
+   * serverless environment and we wanted to make sure debug agent always
+   * captures the snapshots.
+   */
+  async isReady(): Promise<void> {
```

Comment:
`async` is unnecessary if you don't use `await` in the function body.

Reply:
Removed.